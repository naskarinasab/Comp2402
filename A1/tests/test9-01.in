Patrick
Fred
Pad
Patr
Pa
Fr
P
Freddie
Rick
Helpful
Helpfu
Help
Patric
Patrick goes
