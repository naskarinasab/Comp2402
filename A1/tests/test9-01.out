Patrick
Fred
Pad
Freddie
Rick
Helpful
Patrick goes
